### PR TITLE
Initial scope

### PR DESCRIPTION
**Pull request by jonare**
MVP scope with exclusions as described at https://confluence.intern.sparebank1.no/x/9oW-UQ

The file is mainly aimed for use in Burp Suite. This is explained in further details at https://confluence.intern.sparebank1.no/x/ywGDVQ

### CODE REVIEW

File: sb1-scope.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/utsett\\/betaling*",
+          "host": "sparebank1.no",
+          "protocol": "any"
+        }
+      ],
+      "include": [
```
Comment (jonare) [inline sb1-scope.json:169]: host for include will be changed to www.sparebank1.no and mobilbank-pm.sparebank1.no.

File: sb1-scope.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
+{
```
Comment (fossecode) [inline sb1-scope.json:1]: Where is this file used?

Comment (jonare) [reply sb1-scope.json:1]: Apologies for the missing context. It's now explained at https://confluence.intern.sparebank1.no/x/ywGDVQ

**Review by tkill456 — APPROVED** [review]

**Review by fossecode — APPROVED** [review]

**Review by StianHH — APPROVED** [review]

File: sb1-scope.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste\\/ask",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
```
Comment (idachristinek) [inline sb1-scope.json:156]: looks to be a duplicate of line 139-143

**Review by sjurkvammen — APPROVED** [review]

File: sb1-scope.json [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+          "file": "\\/api\\/personal\\/banking\\/credit\\/accounts\\/*\\/signing-orders",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/api\\/personal\\/banking\\/credit\\/refinancing\\/applications*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
```
Comment (marianne-grov) [inline sb1-scope.json:29]:
```suggestion
        },
        {
          "enabled": true,
           "file": "\\/api\\/personal\\/banking\\/membership\\/coop*",
           "host": "^*.sparebank1.no$",
           "protocol": "any"
        },
        {
          "enabled": true,
           "file": "\\/api\\/personal\\/banking\\/studentloan*",
           "host": "^*.sparebank1.no$",
           "protocol": "any"
        },
        {
          "enabled": true,
           "file": "\\/api\\/personal\\/banking\\/credit\\/unsecured-debt*",
           "host": "^*.sparebank1.no$",
           "protocol": "any"
        },
```

File: sb1-scope.json (hunk 3 above)
Comment (jonare) [reply sb1-scope.json:156]: Thanks! e2da58b93d5542a31353b248c9dbf1c1ce242a75

**Review by idachristinek — APPROVED** [review]

**Review by eirikdal — APPROVED** [review]

**Review by rubenesaur — APPROVED** [review]

**Review by eric-shmeric — APPROVED** [review]

**Review by MarkusSN — APPROVED** [review]

**Review by tabmy — APPROVED** [review]

**Pull request merged**